website/docs/integrations/mqtt.md

```diff
@@ -58,6 +58,7 @@ Vehicle data will be published to the following topics:
 | `teslamate/cars/$car_id/plugged_in`                    | true                 | If car is currently plugged into a charger                                            |
 | `teslamate/cars/$car_id/charge_energy_added`           | 5.06                 | Last added energy in kWh                                                              |
 | `teslamate/cars/$car_id/charge_limit_soc`              | 90                   | Charge Limit Configured in Percentage                                                 |
+| `teslamate/cars/$car_id/charge_rate`                   | 0.0                  | Charge rate in km per hour                                                           |
```

Comment:
Since this PR seemed abandoned and I want to have this feature I've redonde some of the changes in this new PR: https://github.com/teslamate-org/teslamate/pull/4130
Can you please review it there? I'm not 100% sure it's correct

Reply:
Charge rate is effectively always in mph. And your discrepancy is not only efficiency, is energy not going into the battery but being used for other things. Charge rate is a very good indicator of how the energy coming from the charge port is being used:
Plugged in using 32A * 240v = 7.68kW, car and API reporting 32A 8kw. Charge rate = 0.... I'm cooling down the cabin at 40ºC to 20ºC, no energy going into the battery! Then later it reports charge rate = 5 mph, this means in my car, 1400W only going into the battery.
This metric is so important and not being used in teslamate! Teslafi uses this to calculate energy spent cooling as you can directly calculate it.